src/util/CMakeLists.txt

```diff
@@ -4,9 +4,9 @@ file (GLOB_RECURSE util_SOURCES *.cpp *.h)
 
 add_library (util STATIC ${util_SOURCES})
 add_dependencies(util std::filesystem)
-target_link_libraries(util PUBLIC std::filesystem cxx17)
-target_compile_features(util PUBLIC ${PROJECT_CXX_FEATURES})
-target_include_directories(util PUBLIC
-        $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>
-        #$<INSTALL_INTERFACE:>  # <prefix>/include/mylib
+target_link_libraries(util
+        PUBLIC xoj::defaults xoj::external_modules std::filesystem cxx17
         )
+target_compile_features(util PUBLIC ${PROJECT_CXX_FEATURES})
+target_include_directories(util PUBLIC $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>)
+add_library(xoj::util ALIAS util)
```

Comment:
Why make this an alias if you could just use the actual name throughout the file?